blog/models.py

```diff
@@ -0,0 +1,13 @@
+from django.utils import timezone
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+class Post(models.Model):
+    title = models.CharField(max_length=100)
```

Comment:
have to max_lenght in the 2 power number

Reply:
Done. to utilise space completely